manim/mobject/types/point_cloud_mobject.py

```diff
@@ -55,19 +67,25 @@ def construct(self):
 
     """
 
-    def __init__(self, stroke_width=DEFAULT_STROKE_WIDTH, **kwargs):
+    def __init__(self, stroke_width: int = DEFAULT_STROKE_WIDTH, **kwargs: Any) -> None:
         self.stroke_width = stroke_width
         super().__init__(**kwargs)
 
-    def reset_points(self):
+    def reset_points(self) -> Self:
         self.rgbas = np.zeros((0, 4))
         self.points = np.zeros((0, 3))
         return self
 
-    def get_array_attrs(self):
+    def get_array_attrs(self) -> list[Any]:
```

Comment:
Shouldn't this be `list[str]` if I remember correctly?

Reply:
Implemented.